uint64 time_started
string topic

float32 mouse_x
float32 mouse_y

uint8 mouse_event
bool  shift_pressed
bool  ctrl_pressed
bool  alt_pressed